src/libs/actions/Report.ts

```diff
@@ -3673,6 +3673,30 @@ function setGroupDraft(newGroupDraft: Partial<NewGroupChatDraft>) {
     Onyx.merge(ONYXKEYS.NEW_GROUP_CHAT_DRAFT, newGroupDraft);
 }
 
+function setTaskDataForNewAssingee(assigneeLogin: string, accountID: number | undefined = undefined) {
+    const assigneeAccountID = accountID ?? UserUtils.generateAccountID(assigneeLogin);
+    const report: OnyxEntry<Report> | undefined = ReportUtils.buildOptimisticChatReport([assigneeAccountID]);
```

Comment:
```suggestion
    const report: ReportUtils.OptimisticChatReport = ReportUtils.buildOptimisticChatReport([assigneeAccountID]);
```
You will also need to extend `ReportUtils.OptimisticChatReport` to accept `isHidden` prop